packages/core/src/m3/builtins.ts

```diff
@@ -1,18 +1,23 @@
 import {LanguageFactory} from "./factory.js"
 import {lioncoreBuiltinsKey, Property} from "./types.js"
-import {concatenator} from "../utils/string-mapping.js"
+import {StringsMapper} from "../utils/string-mapping.js"
```

Comment:
Should this be here?

Reply:
What do you mean? The type of the `lioncoreBuiltinsIdAndKeyGenerator` constant is `StringsMapper`. (I could define that constant without explicitly stating that type, though.)

Reply:
I meant: "Should this be in this PR?"
Not really important, it's the only non-pure renaming change.

Reply:
Ah yeah, now I get it :) Good question. This particular change comes on the back of cleaning up the whole ID and key generation around LionCore M3, which was a bit too “magical”.